src/components/PaymentLog.vue

```diff
@@ -170,7 +170,7 @@
                   v-model="salesLogForm.repayment_cycle_id"
                   v-validate="'required'"
                   @change="customDate($event)"
-                  :disabled="isCashNCarry"
+                  :disabled="isCashNCarry || lockRepaymentCycle"
```

Comment:
What does `lockRepaymentCycle` mean?

Reply:
a prop to disable the dropdown if nobs is selected

Reply:
What are we disabling? we want to use the existing calculator for monthly repayments

Reply:
yes. when it is nobs, it should preselect bi-monthly and lock the dropdown, but if not nobs, it works normally

Reply:
there in no monthly option for altara pay

Reply:
no we dont want to lock it, they should be able to select monthly as well and work as it is currently